docs/src/Dropdown.doc.js

```diff
@@ -98,49 +89,45 @@ card(
         name: 'anchor',
         type: '?HTMLElement',
         description:
-          'Ref for the element that the Dropdown will attach to, will most likely be a Button',
-        href: 'default',
+          'Ref for the element that the Dropdown will attach to, will most likely be a Button. See the [Accessibility](#Accessibility) guidelines to learn more.',
       },
       {
         name: 'children',
         required: true,
         type: 'React.ChildrenArray<React.Element<typeof DropdownItem | typeof DropdownSection>>',
-        description: 'Should consist of Dropdown.Item and/or Dropdown.Section components',
-        href: 'default',
+        description:
+          'Should consist of Dropdown.Item and/or Dropdown.Section components. See the [Types of items](#Types-of-items) variant to learn more.',
```

Comment:
Should is more of a recommendation, should it be "Must be Dropdown.Item and/or Dropdown.Section components"?